src/components/Toaster/index.tsx

```diff
@@ -8,25 +8,26 @@ export const Toast: any = (
   type: ToastProps['type'],
   title: ToastProps['title'],
   message?: ToastProps['message'],
+  position?: ToastProps['position'],
 ) => {
   const messageToShow = message ? titleWithMessage(title, message) : titleWithoutMessage(title)
   let toastToShow
 
   switch (type) {
     case 'error':
-      toast.error(messageToShow)
+      toast.error(messageToShow, { position: position || toast.POSITION.TOP_RIGHT })
```

Comment:
Rather than accounting for an undefined position, I think we should be able to use defaultProps. 

Right above the export statement, we can do:  

```ts
Toast.defaultProps = {
  position: toast.POSITION.TOP_RIGHT,
}
```

Reply:
Changed, can you check if I did right?

Reply:
Seems like you have it right. Can we add some tests and a story in the storybook  to show this new functionality?

Reply:
I think we can, I see there a story with toaster, can modify it and add a different position to every case. ok?

Reply:
But I don't know to write a test to such case :D

Reply:
Sounds like we should add some 😊. 

Here is an example of testing default props: https://github.com/HospitalRun/components/blob/master/test/badge.test.tsx

Reply:
I could not understand how to test a function since I made changes to a function toast that fires up onClick event. Can you please clarify how to do this?

Reply:
We'll open another issue to take care of the missing tests for this pull request!

Reply:
Nice, so, is everything alright with my PR? Is there anything to change?

Reply:
I gave you some bad code. Default Props will not work in this scenario. 

Let's go with an approach like you originally had, but let's extract the logic for figuring out if the component should use the default position or the one passed-in to a variable. 

Something like: 
`const actualPosition = position || toast.POSITION.TOP_RIGHT`

and then...

`toast.error(messageToShow, { position: actualPosition })`

Reply:
Then the function will go back to accepting an optional position

Reply:
Ok, will do